CHANGELOG.rst

```diff
@@ -1,6 +1,13 @@
 Unreleased_
 -----------
 
+Added
+^^^^^
+* `@HiromuHota`_: Add a custom MLflow model that allows you to package a Fonduer model.
```

Comment:
Awesome!

Reply:
Thanks.
Just to be clear, IMO I don't think Fonduer needs to be v0.9.X for this PR since this PR does not introduce any breaking changes.